cmd/roles/roles.go

```diff
@@ -49,11 +49,20 @@ import (
 // all milvus related metrics is in a separate registry
 var Registry *prometheus.Registry
 
+// all Cgo metrics in a separated registry
+var CRegistry *metrics.CRegistry
+
 func init() {
 	Registry = prometheus.NewRegistry()
 	Registry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
 	Registry.MustRegister(prometheus.NewGoCollector())
 	metrics.RegisterEtcdMetrics(Registry)
+
+	CRegistry = &metrics.CRegistry{
+		Registry: prometheus.NewRegistry(),
+	}
+	CRegistry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
+	CRegistry.MustRegister(prometheus.NewGoCollector())
```

Comment:
CRegistry doesn't need go process and go gc collector.

Reply:
ok